README.md

```diff
@@ -100,6 +100,37 @@ Ensure you have "killall" installed on your system (*psmisc* package on Debian-l
 
 If the node and Gateway are on different systems, you may need to utilize the "NOTIFY" endpoint on the Gateway's dashboard/API instead.
 
+## Docker
+
+Clone and build image, Create your config file, see the config_template.json. then:
+
+  ```
+  docker run -d \
+  -v ./YOUR_CONFIG.json:/app/datum_gateway_config.json:ro \
```

Comment:
Any idea how to make this compatible with #49?

Reply:
Currently, #49 uses the best practice of writing a new file and renaming it over the old one. Will that work with this?